src/makielayout_integration.jl

```diff
@@ -1,11 +1,11 @@
 using AbstractPlotting: px, Attributes, AbstractPlot, AbstractPlotting
-using MakieLayout: LAxis,
+using MakieLayout: LAxis, LText, LRect,
                    GridLayout,
-                   linkxaxes!,
-                   linkyaxes!,
-                   hidexdecorations!,
-                   hideydecorations!,
-                   MakieLayout
+                   linkxaxes!, linkyaxes!,
+                   hidexdecorations!, hideydecorations!,
+                   MakieLayout,
+                   Top, Bottom, Left, Right,
+                   RGBAf0, lift, @lift, Node
```

Comment:
where do these actually come from? is it ok to use them from MakieLayout?

Reply:
these are from MakieLayout

```
hidexdecorations!, hideydecorations!,
MakieLayout,
Top, Bottom, Left, Right,
```

`RGBAf0` is from Colors

and the others are from AbstractPlotting

Reply:
Top, Bottom, Left, Right are deprecated, I think.  RGBAf0 is from AbstractPlotting (shortcut for RGBA{Float32} from Colors) and the lift stuff is from AbstractPlotting also.

Reply:
> Top, Bottom, Left, Right are deprecated, I think
I didn't deprecate them, did someone else? Why would they be